Tests/NetworkServiceTests/NetworkErrorTests/NetworkError+LocalizedErrorTests.swift

```diff
@@ -0,0 +1,16 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorLocalizedErrorTests: XCTestCase {
+    func testErrorDescriptionExistsInAPI() {
```

Comment:
Can we use the same approach to verify that all cases are included as we used in `NetworkErrorTests`?
 if we forget to add a new case the test will pass, and we will not catch the potential problem.

Reply:
This method has been removed.